stubs/Drupal/Core/Field/FieldItemList.stub

```diff
@@ -5,16 +5,15 @@ namespace Drupal\Core\Field;
 use Drupal\Core\TypedData\Plugin\DataType\ItemList;
 
 /**
- * @template T of \Drupal\Core\Field\FieldItemInterface
+ * @template T of FieldItemInterface
  * @extends ItemList<T>
  * @implements FieldItemListInterface<T>
  */
 class FieldItemList extends ItemList implements FieldItemListInterface {
 
   /**
-   * @return \Drupal\Core\Field\FieldItemInterface
+   * @return T
    */
-  protected function createItem(int $offset = 0, ?mixed $value = NULL): \Drupal\Core\Field\FieldItemInterface {
-  }
+  protected function createItem(int $offset = 0, mixed $value = NULL): FieldItemInterface {}
```

Comment:
Technically it needs to stay `?mixed`? or does `mixed` imply null?

Reply:
`?mixed` is wrong from the syntax point of view, as it implies `NULL`. I thought it could lead to some issues because this could lead to a wrong AST. But it clearly wasn't an issue, so I can revert it back, no problem. However, it would be better to consider removing the nullable type.

https://www.php.net/manual/en/language.types.mixed.php: 

> The [mixed](https://www.php.net/manual/en/language.types.mixed.php) type accepts every value. It is equivalent to the [union type](https://www.php.net/manual/en/language.types.type-system.php#language.types.type-system.composite.union) `object|resource|array|string|float|int|bool|null`. Available as of PHP 8.0.0.